CONTRIBUTING.md

```diff
@@ -0,0 +1,239 @@
+
+- Open an Issue at [discord-modmail/modmail/issues][modmail-issues].
+- Explain the problem and include additional details to help maintainers reproduce the problem:
+  - **Use a clear and descriptive title** for the issue to identify the problem.
+  - **Describe the exact steps which reproduce the problem** in as many details as possible. When listing steps, **don't just say what you did but explain how you did it**.
+  - **Provide specific examples to demonstrate the steps**. Include links to files or GitHub projects, or copy/paste-able snippets, which you use in those examples. If you're providing snippets in the issue, use [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+  - **Describe the behaviour you observed after following the steps** and point out what exactly is the problem with that behaviour.
+  - **Explain which behaviour you expected to see instead and why.**
+  - **Include screenshots and animated GIFs** which show you following the described steps and clearly demonstrate the problem. If you use the keyboard while following the steps, **record the GIF with the [Keybinding Resolver](https://github.com/atom/keybinding-resolver) shown**. You can use [this tool](https://www.cockos.com/licecap/) to record GIFs on macOS and Windows, and [this tool](https://github.com/colinkeenan/silentcast) on Linux (of course there are plenty more).
+
+## Request a Feature
+
+If the project doesn't do something you need or want it to do:
+
+- Open an Issue at [discord-modmail/modmail/issues][modmail-issues].
+- Provide as much context as you can about what you're running into.
+  - **Use a clear and descriptive title** for the issue to identify the suggestion.
+  - **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+  - **Provide specific examples to demonstrate the steps**. Include copy/paste-able snippets which you use in those examples, as [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+  - **Explain why this enhancement would be useful** to Modmail, and would benefit the community members.
+- Please try and be clear about why existing features and alternatives would not work for you.
+
+Once it's filed:
+
+- The Maintainers will [label the issue](#label-issues).
+- The Maintainers will evaluate the feature request, possibly asking you more questions to understand its purpose and any relevant requirements. If the issue is closed, the team will convey their reasoning and suggest an alternative path forward.
+- If the feature request is accepted, it will be marked for implementation with `status: approved`, which can then be done either by a core team member or by anyone in the community who wants to contribute code.
+
+> **Note**: The team is unlikely to be able to accept every single feature request that is filed. Please understand if they need to say no.
```

Comment:
Mention being able to write a plugin again here. Best to have no chances of missing that option rather than field extra issues.